src/seedu/addressbook/commands/MultiDeleteCommand.java

```diff
@@ -0,0 +1,57 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.common.Messages;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.person.UniquePersonList;
+
+import java.util.ArrayList;
+import static seedu.addressbook.ui.TextUi.DISPLAYED_INDEX_OFFSET;
+
+/**
+ * Deletes multiple people identified using the input which is taken as the
+ * index of the person.
+ */
+public class MultiDeleteCommand extends Command{
+    
+    private ArrayList<Integer> multiIndexStorage;
+    private boolean isValid;
+
+    public static final String COMMAND_WORD = "multidelete";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + "Deletes all persons identified by the index numbers " +
+            "used in the last find/list call.\n" +
+            "Parameters: INDICES\n" + 
+            "Example: " + COMMAND_WORD + " 2 4 5";
+    public static final String MESSAGE_MULTIDELETE_SUCCESS = "Deleted Person: %1$s";
+    public static final String MESSAGE_INVALID_INPUT = "Invalid Input";
+    
+    public MultiDeleteCommand(ArrayList<Integer> listIndex, boolean isValid) {
+        this.multiIndexStorage = listIndex;
+        this.isValid = isValid;
+    }
+    
+    @Override
+    public CommandResult execute() {
+        
+        if(isValid) {
```

Comment:
the bulk of the method which is to do the actual processing should not be indented too much. ie avoid arrow-head style of code.

Instead, can consider:
if (!isValid) {
 //error handling
}

//do work